Monstarlab.EntityFramework.Extension/Repositories/UnitOfWork.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace Monstarlab.EntityFramework.Extension.Repositories;
+
+public class UnitOfWork<TContext> : IUnitOfWork where TContext : DbContext
+{
+    private readonly TContext _context;
+
+    public UnitOfWork(TContext context)
+    {
+        _context = context;
+    }
+
+    public Task CommitAsync(CancellationToken cancellationToken = default)
```

Comment:
Suggestion/personal preference: I would have done it like this:
```suggestion
    public Task CommitAsync(CancellationToken cancellationToken = default) => _context.SaveChangesAsync(cancellationToken);
```